Human.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace School
+{
+    public class Human
+    {
+        public Human()
+        {
+            
+        }
+
+        public Human(string name, string lastName, int ID)
+        {
+            Name = name + " " + lastName;
+            this.ID = ID;
+        }
+
+        protected Human(string name, string lastName, int ID, bool isHumanAtSchool)
```

Comment:
What is that bool used for? And why do we need a separate method that does exactly the same anyway?

Reply:
removed